app/org/sagebionetworks/bridge/dynamodb/DynamoReportIndexDao.java

```diff
@@ -33,8 +34,13 @@ public void addIndex(ReportDataKey key) {
         DynamoReportIndex index = new DynamoReportIndex();
         index.setKey(key.getIndexKeyString());
         index.setIdentifier(key.getIdentifier());
-        
-        mapper.save(index);
+
+        // Don't recreate if it exists. There isn't a non-key attribute to use for a 
+        // conditional save operation, so load, test, then save.
+        DynamoReportIndex dbIndex = mapper.load(index);
+        if (dbIndex == null) {
+            mapper.save(index);
+        }
```

Comment:
Found this and added. For addIndex we do want to silently return if index exists, so this is now swallowing the exception.